packages/babel-plugin-emotion/src/index.js

```diff
@@ -228,13 +228,50 @@ export function replaceCssObjectCallExpression(path, identifier, t) {
 
 const visited = Symbol('visited')
 
+const parseImports = (node, currentNames) => {
+  const names = Object.assign({}, currentNames)
+  const imports = node.specifiers.filter(
+    specifier =>
+      specifier.type === 'ImportDefaultSpecifier' ||
+      specifier.type === 'ImportSpecifier'
+  )
+
+  imports.forEach(singleImport => {
+    if (singleImport.imported) {
+      // Is helper method
+      names[singleImport.imported.name] = singleImport.local.name
+    } else {
+      // Is default import
+      names.default = singleImport.local.name
+    }
+  })
+
+  return names
+}
 export default function(babel) {
   const { types: t } = babel
 
+  let importedNames = {
+    default: 'styled',
+    css: 'css',
+    keyframes: 'keyframes',
+    injectGlobal: 'injectGlobal'
+  }
+
   return {
     name: 'emotion', // not required
     inherits: require('babel-plugin-syntax-jsx'),
     visitor: {
+      ImportDeclaration: {
+        enter({ node }) {
+          if (node.source.value === 'emotion') {
```

Comment:
I think this should be `node.source.value.indexOf('emotion') !== -1` so it includes emotion, react-emotion, preact-emotion and any other runtimes people create.